src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -0,0 +1,224 @@
+package com.minecolonies.blocks;
+
+import com.minecolonies.MineColonies;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.creativetab.ModCreativeTabs;
+import com.minecolonies.inventory.InventoryField;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import com.minecolonies.util.EntityUtils;
+import com.minecolonies.util.LanguageHandler;
+import net.minecraft.block.BlockContainer;
+import net.minecraft.block.ITileEntityProvider;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.block.ITileEntityProvider'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)